flutter/lib/sentry_flutter.dart

```diff
@@ -76,6 +77,10 @@ mixin SentryFlutter {
     }
     options.addIntegration(flutterErrorIntegration);
     options.addIntegration(nativeSdkIntegration(options, _channel));
+
+    if (Platform.isIOS) {
```

Comment:
we probably need to make it testable, so using this static might need to change, look what I've done with the `PackageLoader` typedef.
We can improve this later, of course.

Reply:
added a Todo. for now I should be able to test it with `onPlatform`